client/ayon_core/pipeline/create/context.py

```diff
@@ -1592,6 +1604,12 @@ def get_current_project_anatomy(self):
                 self._current_project_name)
         return self._current_project_anatomy
 
+    def get_current_project_settings(self):
```

Comment:
This method should be used in `_reset_creator_plugins`.

Reply:
Done